spec/index.html

```diff
@@ -733,125 +733,140 @@ <h3>Literals</h3>
     <p>Literals are used for values such as strings, numbers, and dates.</p>
 
     <p>A <dfn data-local-lt="RDF literal">literal</dfn> in an <a>RDF graph</a> consists of
-      two, three, or four elements, as follow:</p>
+      two, three, or four elements, as follow.</p>
 
     <ol>
-      <li>a <dfn>lexical form</dfn> consisting of a sequence of
-        <a data-cite="I18N-GLOSSARY#dfn-code-point" class="lint-ignore">Unicode code points</a> [[!UNICODE]]
-        which are <a data-cite="I18N-GLOSSARY#dfn-scalar-value">Unicode scalar values</a>,
-        and therefore do not contain
-        <a data-cite="I18N-GLOSSARY#dfn-surrogate" class="lint-ignore">Unicode surrogate code points</a></li>
-      <li>a <dfn>datatype IRI</dfn>, being an <a>IRI</a>
+      <li>A <dfn>lexical form</dfn>, being an [=RDF string=].
+      <li>A <dfn>datatype IRI</dfn>, being an <a>IRI</a>
         identifying a datatype that determines how the lexical form maps
-        to a <a>literal value</a></li>
-      <li>if and only if the <a>datatype IRI</a> is
-        <code>http://www.w3.org/1999/02/22-rdf-syntax-ns#langString</code>, a
+        to a <a>literal value</a>.</li>
+      <li>If and only if the <a>datatype IRI</a> is
+        <code>http://www.w3.org/1999/02/22-rdf-syntax-ns#langString</code> or
+        <code>http://www.w3.org/1999/02/22-rdf-syntax-ns#dirLangString</code>, a
         non-empty <dfn>language tag</dfn> as defined by [[!BCP47]]. The
         language tag MUST be well-formed according to
         <a data-cite="bcp47#section-2.2.9">section 2.2.9</a>
         of [[!BCP47]],
         and MUST be treated consistently, that is, in a case insensitive manner.
```

Comment:
```suggestion
        and MUST be treated consistently in a case insensitive manner.
```

Reply:
I think we are losing something with this change. Namely, the idea is that they "MUST be treated consistently [with BCP47]". With the proposed wording, "consistently" seems to only mean "always the same way".

Reply:
> The that is implies that consistent treatment is equivalent to case insensitive treatment.

Again, since [BCP47] is very shortly before the word "consistently", this equivalence must be understood as "consistent *with BCP47*"  and "case insensitive". At least that's how I understand this sentence (which, I now realize, was not even written by me :).

Would it work better if we replaced "consistently" with "accordingly"?

```suggestion
        and MUST be treated accordingly, that is, in a case insensitive manner.
```